app/views/admin/loans/_form.html.slim

```diff
@@ -16,9 +16,10 @@
   = authorized_form_field(simple_form: f, model: @loan, field_name: :organization,
     choices: @organization_choices, include_blank_choice: false, classes: 'hidden-print')
 
-  / = f.input :name
-  /   .view-element = @loan.name
-  /   = f.input_field :name
+  = f.input :name
+    .view-element = @loan.id ? @loan.display_name : ''
```

Comment:
The proper way to check if an object is persisted is to use `.persisted?` or `.new_record?` (the opposite). Checking `id` works but is less clear.